geometric/params.py

```diff
@@ -327,16 +371,16 @@ def parse_optimizer_args(*args):
     grp_optparam.add_argument('--conmethod', type=int, help='Set to 1 to enable updated constraint algorithm (default 0).\n ')
     grp_optparam.add_argument('--reset', type=str2bool, help='Reset approximate Hessian to guess when eigenvalues are under epsilon.\n '
                               'Defaults to True for minimization and False for transition states.\n ')
+    grp_optparam.add_argument('--skip', action='store_true', help='Skip Hessian updates that would introduce negative eigenvalues.')
```

Comment:
The name of this option is too vague.  We should brainstorm a different name that is more specific but also not overly long.  Maybe `--verify-h-updates`?

Reply:
Is this option used or important when running NEB calculations?

Reply:
NEB does not use this option. I removed it along with some other parameters.